java/ql/test/kotlin/library-tests/reflection/reflection.expected

```diff
@@ -263,29 +263,78 @@ modifiers
-| reflection.kt:83:17:83:28 | getValue | 3 |
-| reflection.kt:105:18:105:31 | getProp1 | 3 |
-| reflection.kt:105:18:105:31 | setProp1 | 3 |
-| reflection.kt:126:9:126:13 |  | 1 |
-| reflection.kt:131:1:131:50 | takesOptionalParam$default | 10 |
-| reflection.kt:134:21:134:40 |  | 1 |
-| reflection.kt:140:5:140:54 | takesOptionalParam$default | 10 |
-| reflection.kt:144:21:144:41 |  | 1 |
-| reflection.kt:145:32:145:70 |  | 1 |
-| reflection.kt:150:1:150:60 | extTakesOptionalParam$default | 10 |
-| reflection.kt:153:21:153:44 |  | 1 |
-| reflection.kt:154:33:154:61 |  | 1 |
-| reflection.kt:157:1:157:49 | ConstructorOptional | 10 |
-| reflection.kt:162:25:162:45 |  | 1 |
+| file://<external>/CharIterator.class:0:0:0:0 | forEachRemaining | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/CharProgression.class:0:0:0:0 | forEach | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/CharProgression.class:0:0:0:0 | spliterator | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/CharRange.class:0:0:0:0 | forEach | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/CharRange.class:0:0:0:0 | spliterator | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/Class2.class:0:0:0:0 | getValue | Default property accessor |
+| file://<external>/Class2.class:0:0:0:0 | getValue | Default property accessor |
+| file://<external>/Collection.class:0:0:0:0 | forEach | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/IntIterator.class:0:0:0:0 | forEachRemaining | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/IntProgression.class:0:0:0:0 | forEach | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/IntProgression.class:0:0:0:0 | spliterator | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/IntRange.class:0:0:0:0 | forEach | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/IntRange.class:0:0:0:0 | spliterator | Forwarder for a Kotlin class inheriting an interface default method |
+| file://<external>/KTypeProjection.class:0:0:0:0 | contravariant | Proxy static method for a @JvmStatic-annotated function or property |
+| file://<external>/KTypeProjection.class:0:0:0:0 | copy$default | Forwarder for Kotlin calls that need default arguments filling in |
+| file://<external>/KTypeProjection.class:0:0:0:0 | covariant | Proxy static method for a @JvmStatic-annotated function or property |
+| file://<external>/KTypeProjection.class:0:0:0:0 | invariant | Proxy static method for a @JvmStatic-annotated function or property |
+| file://<external>/List.class:0:0:0:0 | forEach | Forwarder for a Kotlin class inheriting an interface default method |
```

Comment:
Looks like this is the Kotlin Iterable equivalent of https://docs.oracle.com/javase/8/docs/api/java/lang/Iterable.html#forEach-java.util.function.Consumer- -- there's also the extension function defined in CollectionsKt, which takes a `Function1<T, Unit>` instead of a `Consumer<T>`. Not sure why they bothered to do that instead of just SAM converting the function to Consumer.